b2sdk/sync/exception.py

```diff
@@ -35,3 +34,25 @@ def __str__(self):
             self.filename,
             self.encoding,
         )
+
+
+class InvalidArgument(B2Error):
+    """
+    Raised when one or more arguments are invalid
+    """
+
+    def __init__(self, field_name, message):
```

Comment:
I think that technically, this should be `parameter_name`, not `field_name`.  A parameter is something passed into  a function, and a field is a slot in an object.